projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/qualifying-questions/qualifying-question-options/qualifying-question-options.component.ts

```diff
@@ -8,10 +9,24 @@ import { QualifyingQuestion } from '../../../models';
   selector: 'ccd-qualifying-question-options',
   templateUrl: './qualifying-question-options.component.html'
 })
-export class QualifyingQuestionOptionsComponent {
+export class QualifyingQuestionOptionsComponent implements OnInit{
   @Input() public qualifyingQuestionsControl: FormControl;
   @Input() public qualifyingQuestions$: Observable<QualifyingQuestion[]>;
   public qualifyingQuestionsErrorMessage = QualifyingQuestionsErrorMessage;
+  public caseId: string;
+
+  constructor(
+    private readonly route: ActivatedRoute,
+    private readonly router: Router) {
+  }
+
+  public ngOnInit(): void {
+    this.caseId = this.route.snapshot.params.cid;
+  }
+
+  public click():void {
```

Comment:
@DavidJayakumar - really minor one; spacing between : and `void`

Reply:
done